notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -398,9 +398,14 @@
         "3. In the **Service account name** field, enter a name, and\n",
```

Comment:
<p>add some explanation of the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>done</p>